python/lsst/analysis/ap/apdb.py

```diff
@@ -66,13 +78,65 @@ def connection(self):
         """
         pass
 
-    def load_sources_for_object(self, dia_object_id, limit=100000):
+    def set_bad_DiaSource_flags(self, flag_list):
+        """Set DiaSource flags to exclude.
```

Comment:
"to exclude from `load_sources`?" - I'm having a bit of trouble wording this in a way that makes it very explicit how these are used.